docs/getting-started/introduction.md

```diff
@@ -50,6 +49,73 @@ class CounterExampleState(rx.State):
     def decrement(self):
         self.count -= 1
 
+class IntroTabsState(rx.State):
+    """The app state."""
+
+    value = "tab1"
+    tab_selected = ""
+
+    def change_value(self, val):
+        self.tab_selected = f"{val} clicked!"
+        self.value = val
+
+def tabs():
+    return rx.tabs.root(
+        rx.tabs.list(
+            rx.tabs.trigger(
+                "Frontend", value="tab1"
+            ),
+            rx.tabs.trigger(
+                "Backend", value="tab2"
+            ),
+            rx.tabs.trigger(
+                "Page", value="tab3"
+            ),
+        ),
+        rx.tabs.content(
+            rx.markdown(
+                """The frontend is built decratively using Reflex components. These component can be nested and customized to create complex layouts and interactions. More information can be found in the [Reflex Components](/docs/components) section.- Use Reflex components and var operations
```

Comment:
"- Use Reflex components and var operations" needs to be on a new line, doesn't seem to show up as a bullet point